policy/modules/services/openvpn.te

```diff
@@ -67,6 +67,8 @@ allow openvpn_t openvpn_etc_t:dir list_dir_perms;
 allow openvpn_t openvpn_etc_t:file read_file_perms;
 allow openvpn_t openvpn_etc_t:lnk_file read_lnk_file_perms;
 
+allow openvpn_t openvpn_runtime_t:sock_file { create unlink };
```

Comment:
The `create` and `unlink` permissions should probably be replaced by `manage_sock_file_perms`.
This could also be moved down to the other `manage_..._pattern` calls and be replaced by `manage_sock_files_pattern(openvpn_t, openvpn_runtime_t, openvpn_runtime_t)`
Since this states the intent more clearly: Managing a socket file that is labeled `openvpn_runtime_t` and is inside a directory labeled `openvpn_runtime_t`